src/frontend/src/handler/alter_table_column.rs

```diff
@@ -145,16 +150,17 @@ pub async fn handle_alter_table_column(
         }
     }
 
+    let mut columns_altered = original_catalog.columns.clone();
```

Comment:
Can you elaborate why we need this?

Reply:
For cdc table created by `create table (*)` syntax, column defs is empty.